scripts/new_version.py

```diff
@@ -25,37 +25,33 @@ def main(argv):
     else:
         dev_version = re.compile(r'dev([\d]+)').search(version).group(1)
         new_dev_version = int(dev_version) + 1
-        new_version = version.replace("dev%s" % dev_version, "dev%s" % new_dev_version)
+        new_version = version.replace(f"dev{dev_version}", f"dev{new_dev_version}")
```

Comment:
This script has been copy-pasted between various projects, do we need to correct this in other places as well ?

Reply:
It's not really a correction, I just modernised the code with pyupgrade now that we support Python >=3.6 only.
I think updating these scripts together with the rest of each project is OK, but we could surely update the "source" from where they are copied from, if there's one (@jmchilton should know).

Reply:
Who knows 😆 - I just copied it from whatever project I had last touched probably. 

Thanks for the updates and fixes.